mp4/mime.go

```diff
@@ -31,6 +32,9 @@ func DecodeMimeSR(hdr BoxHeader, startPos uint64, sr bits.SliceReader) (Box, err
 		Version: byte(versionAndFlags >> 24),
 		Flags:   versionAndFlags & flagsMask,
 	}
+	if hdr.payloadLen() < 5 {
+		return nil, fmt.Errorf("mime: box size %d less than 5", hdr.Size)
+	}
```

Comment:
This is not the box length, but the payload length. Since the header size can be bigger than 8 bytes, I'd suggest that you replace "box size" with "box payload size".

Reply:
I've updated the error message.